api/database/migrations/2025_08_21_143842_add_contact_email_to_pools.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('pools', function (Blueprint $table) {
+            $table->string('contact_email')->nullable();
```

Comment:
After backfilling, this field should probably be made nonnullable.  That would let you make it non-nullable when querying, too.

Reply:
Changed in [Make contact email non-nullable](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14445/commits/766af39d192f1aba41ab31992f42fa6e4901eca4)